scripts/Dockerfile.base

```diff
@@ -25,6 +22,16 @@ RUN apk add --no-cache \
 		-u 1000 \
 		-G coder \
 		coder
+# Terraform was disabled in the edge repo due to a build issue.
+# https://gitlab.alpinelinux.org/alpine/aports/-/commit/f3e263d94cfac02d594bef83790c280e045eba35
+# Using wget for now. Note that busybox unzip doesn't support streaming.
+ARG ARCH
+RUN wget -O /tmp/terraform.zip "https://releases.hashicorp.com/terraform/1.5.6/terraform_1.5.6_linux_${ARCH}.zip" && \
+    busybox unzip /tmp/terraform.zip -d / && \
+		mv /terraform /usr/local/bin/terraform && \
+		rm -f /tmp/terraform.zip && \
+		chmod +x /usr/local/bin/terraform && \
+		terraform --version
```

Comment:
Why we are checking version?

Reply:
Guessing, but this is a common pattern to verify that it works (and the build output will include a hint too).

Reply:
It's not strictly necessary, but I like to do this as the version shows up in the output and also makes sure the file is executable.